### DISCUSSION TITLE
Multiple Organisations

### DISCUSSION THREAD

**Discussion by LukeyJayT3** [sanderkl/PSMeraki#30] [Q&A] [answered]
Hi There,
Wondering if this can be used with Multiple organisations. When running a command it seems to randomly pick one.
Any ideas.
Thanks

**Comment by sanderkl** [comment, marked as answer]
yes, 
All relevant cmdlets support an -OrgId parameter, so its possible to query multiple orgs in a single script. 
If no OrgID is specified it will default to the first organization from the list "Get-MrkOrganization" produces. 
That same cmdlet can be used to find the OrgID number for each organization you have access to.
